CHANGES.txt

```diff
@@ -22,4 +22,5 @@ Fixes
 =====
 
 - Fixed a bug which could lead to invalid version numbers being transferred to
-  the replica of an item.
+  the replica of an item when the version number increased beyond the maximum
+  value of an integer.
```

Comment:
can we phrase this a bit more user friendly? like "Fixed a streaming issue which could result in wrong version numbers of items on a replica shard."